### PR TITLE
chore: Remove deliveries

### PR DESCRIPTION
**Pull request by brunogaspar**
This removes the deliveries in favor of a more flexible way to be implemented on the application instead.

### CODE REVIEW

File: src/DataExportServiceProvider.php [hunk 1]
```diff
@@ -54,7 +52,7 @@ private function registerGraphQLTypes(TypeRegistry $typeRegistry): void
     {
         $types = [
             ExportResponseStatus::class,
-            DeliveryType::class,
+            DeliveryType::class, // TODO: This should not be part of the package...
```
Comment (brunogaspar) [inline src/DataExportServiceProvider.php:55]: We'll need to take care of this :)

File: src/Services/UpdateExport.php [hunk 2]
```diff
@@ -28,6 +29,8 @@ public function run(): Export
 
         $export->save();
 
+        event(new ExportCompleted($export));
```
Comment (brunogaspar) [inline src/Services/UpdateExport.php:32]: I've moved this from the app to here, seems to make more sense to dispatch the event from the package.

What do you think?

Comment (lukasjuhas) [reply src/Services/UpdateExport.php:32]: agreed!

**Review by lukasjuhas — APPROVED** [review]
👌

**Pull request merged**